libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -586,6 +586,14 @@ bool AP_AHRS_NavEKF::set_origin(const Location &loc)
     case EKF_TYPE3:
         return ret3;
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    case EKF_TYPE_SITL: {
+        struct SITL::sitl_fdm &fdm = _sitl->state;
+        fdm.home = loc;
```

Comment:
This doesn't look good, as on the next call on fill_fdm() the home value will be changed by the starting one stored in sim_aircraft, no ?